src/meta/protos/proto/user.proto

```diff
@@ -194,6 +194,8 @@ message UserStageInfo {
     OnErrorMode on_error = 1;
     uint64 size_limit = 2;
     bool purge = 3;
+    bool single = 4;
```

Comment:
By default `single` is false if it is absent. Does it matter?

Reply:
Yes, single default to false.

Reply:
Yes, it's  false by default

Reply:
I mean, does being `false` have any unexpected side effect? E.g. all uploaded will be multiparted, is that OK?